src/sage/schemes/projective/projective_subscheme.py

```diff
@@ -1404,3 +1404,81 @@ def Chow_form(self):
         rel2 = rel + [CF]
         assert all(f in rel2 for f in CH.gens()), "did not find a principal generator"
         return alp(CF)
+
+    def global_height(self, prec=None):
+        """
+        Return the (projective) global height of the subscheme.
+        INPUT:
+        - ``prec`` -- desired floating point precision (default:
+          default RealField precision).
+        OUTPUT:
+        - a real number.
+        EXAMPLES::
```

Comment:
There are no spaces separating parts of the docstring here and below

Reply:
The standard in

https://doc.sagemath.org/html/en/developer/coding_basics.html#template

(also many examples in the file being modified) suggests blank lines just below `INPUT:` and `OUTPUT:`.